test/specs/reference/tests/05.reference_values.js

```diff
@@ -50,9 +52,159 @@ exports.execute = function (options) {
             return url;
         };
 
+        var moment = options.ermRest._currDate;
+        var expectedMomentValue = "<p>" + moment.day + " " + moment.date + "/" + moment.month + "/" + moment.year + "</p>\n";
+        var testObjects ={
```

Comment:
This whole spec has got to the point that I feel like we should just refactor it later. Just to help with readability of it, can you please add a note in those lines of `expectedValue` that their actual value will be attached with the `setLinkRID` function?